src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
 | `--natt-port` `<int>`            | IPsec NAT Traversal port (default 4500)
 | `--vxlan-port` `<int>`           | Internal VXLAN port (default 4800)
 
+#### `prepare rhos`
+
+`subctl cloud prepare rhos [flags]`
+
+This command prepares an OpenShift installer-provisioned infrastructure (IPI) on Openstack cloud for Submariner installation.
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--cloud-entry` `<string>`       | the cloud entry to use (default `openstack`)
```

Comment:
```suggestion
| `--cloud-entry` `<string>`       | the cloud entry to use (default `openstack`)
```

I know this is copied from subctl, but...as someone reading this for the first time, I'm left wondering: what does "entry" mean in this context? Maybe we can use a different word or way of describing it in the Description, something that helps clarify what it means more than just repeating the words in the flag. Non-blocker for the docs as they are just copying subctl I think.

Reply:
Done.